index.php

```diff
@@ -2347,6 +2347,22 @@ function ProcessChildrenOf($boxparent, &$sql, &$numthispage, &$secondcol, &$thir
 		}
 		$dvd['format'] .= ' ' . $colours;
 
+		if ($dvd['format'] != '') {
+			$dvd['format'] .= '';
+		}
+
+		$dynamicRange = ''
+			. (($dvd['drhdr10']==1)	? "($lang[HDR10])": '')
+			. (($dvd['drdolbyvision']==1)	? "($lang[DOLBYVISION])": '');
+
+		if ($dvd['drhdr10']==1 && $dvd['drdolbyvision']==1) {
+			$dynamicRange = "($lang[HDR10], $lang[DOLBYVISION])";
+		}
+		if ($dynamicRange == '') {
```

Comment:
If `$dynamicRange` is empty, there's no need to concatenate it to `$dvd['format']` (line 2362 is in fact only adding two empty strings to `$dvd['format']`

Reply:
Good catch, I was going to pull this out into a helper, as it felt really dirty.